aemanalyser-maven-plugin/pom.xml

```diff
@@ -219,6 +219,21 @@ governing permissions and limitations under the License.
       <artifactId>org.apache.sling.feature.analyser</artifactId>
       <version>1.5.2</version>
     </dependency>
+    <dependency>
```

Comment:
Why are these dependencies needed here? They are already part of the core module

Reply:
The problem is that all dependencies in feature analyser are set to provided. 
The core module only depends on org.apache.sling.feature.analyser.
How about moving those transitive dependencies to aemanalyser core?

Reply:
Wait, these dependencies are already in this pom, look some lines above

Reply:
You are right. I can simply remove may addition there.